iceoryx_utils/include/iceoryx_utils/internal/concurrent/periodic_task.hpp

```diff
@@ -33,13 +33,20 @@ template <typename T>
 class PeriodicTask
 {
   public:
+    /// @brief Creates a periodic task. The specified callable is stored but not executed.
+    /// To run the task, `void start(const units::Duration interval)` must be called.
+    /// @param[in] taskName will be set as thread name
```

Comment:
Additionally comment needed:
`/// @tparam <template-parameter-name>` for template parameters